crashlogger.js

```diff
@@ -33,9 +45,13 @@ exports = module.exports = function (err, description, data) {
 	out.on("open", fd => {
 		out.write(`\n${stack}\n`);
 		out.end();
-	}).on("error", err => {
-		console.error(`\nSUBCRASH: ${err.stack}\n`);
-	});
+	}).on(
+		"error",
+		/** @param {Error} err */
+		err => {
+			console.error(`\nSUBCRASH: ${err.stack}\n`);
+		}
+	);
```

Comment:
It's probably cleaner to make this a named function in the module scope.
Inline JSDoc is just ugly... (Is this supposed to be better code?)